From bf888184f8857b03f53bba84acb12f29f1068ea9 Mon Sep 17 00:00:00 2001
From: Andrea Merello <andrea.merello@gmail.com>
Date: Sat, 15 Mar 2014 18:29:35 +0100
Subject: [PATCH 10/36] rtl8180: remove too-early-added rtl8187se enum value

While changing board-type variable to enum, I have added enum
value for rtl8187se by mistake.
This will causes gcc warnings with unhandled switch/cases.

Remove it temporarily until I will push also rtl8187se changes.

Signed-off-by: Andrea Merello <andrea.merello@gmail.com>
Signed-off-by: John W. Linville <linville@tuxdriver.com>
---
 drivers/net/wireless/rtl818x/rtl8180/rtl8180.h | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/wireless/rtl818x/rtl8180/rtl8180.h
+++ b/drivers/net/wireless/rtl818x/rtl8180/rtl8180.h
@@ -85,7 +85,6 @@ struct rtl8180_priv {
 	enum {
 		RTL818X_CHIP_FAMILY_RTL8180,
 		RTL818X_CHIP_FAMILY_RTL8185,
-		RTL818X_CHIP_FAMILY_RTL8187SE
 	} chip_family;
 	u32 anaparam;
 	u16 rfparam;
